app/src/androidTest/java/com/android/voyageur/ui/components/PlaceSearchWidgetTest.kt

```diff
@@ -91,30 +91,6 @@ class PlaceSearchWidgetTest {
     composeTestRule.onNodeWithTag("searchTextField").assertIsDisplayed()
   }
 
-  @Test
```

Comment:
nit: mention somewhere in the PR description that you removed this test because it was flaky

Reply:
After merging the changes from main, I see that someone fixed the test, so I re-added it fixed.